CHANGELOG.md

```diff
@@ -17,10 +17,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Changed
 
-- hotfix
```

Comment:
smh

Reply:
lmao

Reply:
tbf it made the v bump a little early so wasn't 100% sure what all was going to be in it 😅